bootstrap.json

```diff
@@ -1,7 +1,11 @@
 {
+    "post-deps": {
+        "golang.org/x/tools/go/internal/gcimporter": "02fcd6aaf1349bf6a3f24ef1b62ca07ff618ff59",
+        "golang.org/x/tools/go/ast/astutil": "02fcd6aaf1349bf6a3f24ef1b62ca07ff618ff59"
```

Comment:
yes we can remove one of these, but since this is a temporary fix, let's just merge this quickly 🙏 

I do believe we should keep the mechanism in place, should be handy in the future.